docs/book/src/topics/managedcluster.md

```diff
@@ -717,7 +719,12 @@ Following is the list of immutable fields for managed clusters:
 
 | CRD                       | jsonPath                     | Comment                   |
 |---------------------------|------------------------------|---------------------------|
+| AzureManagedCluster       | .spec.controlPlaneEndpoint   | populated by the AKS API during cluster create |
+| AzureManagedCluster       | .name                        |                           |
```

Comment:
I think it's fair to omit the `metadata.*` fields here. If CAPZ had some additional requirements on top of what's common to all Kubernetes resources then I think that would be good to include here though. Otherwise there would be a lot more like `uid` and `creationTimestamp` if we wanted this list to be exhaustive and might seem like noise more than being helpful to me at least.

Reply:
+1

Reply:
thx, done